node/CHANGELOG.md

```diff
@@ -42,8 +42,6 @@ All notable changes to this project will be documented in this file.  The format
 * Updated dependencies, in particular `casper-types` to use fixed checksummed-hex format.
 * Add new event to the main SSE server stream across all endpoints `<IP:PORT>/events/*` which emits a shutdown event when the node shuts down.
 * Add `SIGUSR2` signal handling to dump the queue in JSON format (see "Changed" section for `SIGUSR1`).
-
-### Changed
 * `SIGUSR1` now only dumps the queue in the debug text format.
```

Comment:
I believe these 3 points belong in the "unreleased" section, with the first 2 being "added".

Reply:
Updated.
The items in the changelog will require some special care during the merge of fast-sync feature back to dev.